src/main/java/org/example/P11_QuotientAndReminder.java

```diff
@@ -0,0 +1,15 @@
+package org.example;
+import java.util.Scanner;
+
+public class P11_QuotientAndReminder {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter your first number: ");
+        int dividend = scanner.nextInt();
+        System.out.println("Enter your divizor: ");
+       int divizor = scanner.nextInt();
```

Comment:
Always format your code. Use Ctrl+Alt+L in IntelliJ to auto-fromat.

Reply:
Done